MediaBrowser.Providers/Plugins/AudioDb/AudioDbAlbumProvider.cs

```diff
@@ -20,6 +20,9 @@
 
 namespace MediaBrowser.Providers.Plugins.AudioDb
 {
+    /// <summary>
+    /// TheAdioDb Album Provider.
```

Comment:
```suggestion
    /// TheAdioDb album provider.
```

Reply:
I also think it's actually spelled AudioDb 😅

Reply:
```suggestion
    /// TheAudioDb Album Provider.
```